Sdk.zig

```diff
@@ -955,7 +956,8 @@ pub fn signApk(sdk: Sdk, apk_file: []const u8, key_store: KeyStore) *Step {
         key_store.file,
         "--ks-pass",
         pass,
-        sdk.b.pathFromRoot(apk_file),
+        // omit full path on Windows to avoid problems with spaces in the path
```

Comment:
I guess you can mark this with `// HACK: ` as it's not quite correct, but makes things work. It's not a Zig problem, but definitly an android sdk problem (it's missing the quoting in the batch file)

Maybe we can do an ultra-dirty hack for windows:
```zig
if (… windows) b.fmt("\"{s}\"", .{sdk.pathFromRoot(…)}) else …
```
which will include the quotes inside the argument, and thus `%1` will expand to `"…"` instead of `…`. that's definitly a hack tho, but you could also try this with the full path

Reply:
I had actually tried this already and it didn't work, and I couldn't really figure out why. If I type out the command manually and run it in Command Prompt it works, but in PowerShell it fails. And Zig gives me the same error as PowerShell.

Reply:
Added `HACK:`